File: java/FFor.java

public class FFor {

    public static void main (String [] args) {
        ffor(0);
    }

    private static void ffor(int x) {
        for (;;) {
            System.out.println(String.format("x: %d",x));
            x++;
        }
    }

}

File: c/ffor.c
// $ gcc ffor.c

#include <stdio.h>
#include <stdint.h>

void ffor (int x) {
  for (;;) {
    char buff [0x10000];
    printf("x: %d\n",x);
    x ++;
  }
}


int main (int argc, char argv []) {
  ffor (0);
}
